.ci/azure-pipelines-build.yml

```diff
@@ -7,26 +7,26 @@ jobs:
 
     steps:
       - task: NodeTool@0
-        displayName: 'Install Node'
+        displayName: 'Install node'
```

Comment:
We use TitleCase in every other project, it'd probably be good to keep that convention here, for consistency.